Makefile.prow

```diff
@@ -23,6 +23,12 @@ GOSRC := $(GOPATH)/src
 OADP_E2E_DIR := /tmp/oadp-operator
 OADP_E2E_BRANCH := master
 VELERO_IMAGE ?= quay.io/konveyor/velero:latest
+LOCAL_BUILT_IMAGE=ttl.sh/velero-$(shell git rev-parse --short HEAD):1h
```

Comment:
```suggestion
LOCAL_BUILT_IMAGE=ttl.sh/velero-$(shell git rev-parse --short HEAD)-$CLUSTER_ARCH:1h
```

These images are built arch specific.

Reply:
ie. if you run on one config, then switch cluster, you might inadvertedly push image to the first cluster using wrong arch.

Reply:
good suggestion, thank you sir